website/docs/docs/dbt-cloud-environments.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "dbt Cloud environments"
+id: "dbt-cloud-environments"
+description: "Learn about dbt Cloud's development environment to execute your project in the IDE"
+---
+
+An environment determines how dbt Cloud will execute your project in both the dbt Cloud IDE (for development) and scheduled jobs (for deployment).
+
+Critically, in order to execute dbt, environments define three variables:
+
+1. The version of dbt Core that will be used to run your project
+2. The warehouse connection information (including the target database/schema settings)
+3. The version of your code to execute
+
+Each dbt Cloud project can have only one [development environment](#create-a-development-environment), but there is no limit to the number of [deployment environments](/docs/deploy/deploy-environments), providing you the flexibility and customization to tailor the execution of scheduled jobs. 
+
+Use environments to customize settings for different stages of your project and streamline the execution process by using software engineering principles. This page will detail the different types of environments and how to intuitively configure your development environment in dbt Cloud. 
```

Comment:
Might be worth having a link in the future that leads to a software engineering principles page we (dbt) like